assets/Script/game/fallingBlock.js

```diff
@@ -109,13 +109,21 @@ cc.Class({
   },
 
   update(dt) {
-    if (this.movingVertical) {
```

Comment:
This line will need to be re-added.
Essentially, when the block is being dragged the falling animation will stop.